.env.stable

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-675c303
+PFCP_AGENT_IMAGE=ghcr.io/omec-project/upf-epc/upf-epc-pfcpiface:PR-461
```

Comment:
We should update this before merging, to the right commit SHA after https://github.com/omec-project/upf-epc/pull/461 will be merged.

Reply:
@osinstom